src/commands/test/ab/finish.ts

```diff
@@ -0,0 +1,22 @@
+import { flags as oclifFlags } from '@oclif/command'
```

Comment:
Unfortunately `ab` commands are under `workspace` topic in the current version

Reply:
Maybe we could just add them with the alias:
`workspace:ab:finish`
And maintain them under `test`

Reply:
Hum... But then the doc you be weird (since it would still be showed inside the test topic). I think it is better to change to workspace command then :/